DNN Platform/DotNetNuke.Abstractions/Entities/Controllers/IHostController.cs

```diff
@@ -0,0 +1,165 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+
+namespace DotNetNuke.Abstractions.Entities.Controllers
```

Comment:
What if this interface was `DotNetNuke.Abstractions.IHostSettingsController` or `DotNetNuke.Abstractions.Settings.IHostSettingsController`?  Using `Entities.Controllers` doesn't mean anything to me, I'd like to simplify if we can (and set ourselves up with good patterns we want to continue).  Also, `IHostController` doesn't (to me) communicate the scope of the interface, I think `IHostSettingsController` clearly captures the responsibilities in this interface.

Reply:
And, as an added bonus, if we use `IHostSettingsController` as the interface name, we don't need to introduce the `using` alias between the old and new interface.

Reply:
❤❤❤

This makes sense to me and it will allow us to remove all the aliases.

Reply:
I was thinking about this a little bit more. As we migrate and shift interfaces to the abstractions project we should use the opportunity to standardize with naming conventions that are used throughout the rest of the .NET Ecosystem. 

Let's update the `IHostController` -> `IHostSettingsService`

Reply:
Added in latest code push, resolving conversation